README.md

```diff
@@ -1,15 +1,31 @@
-# OpenSearch
+<img src="https://github.com/opensearch-project/documentation-website/blob/main/assets/images/logo.svg" height="64px"/>
 
-Hello!
+We believe that open source makes the world a better place. Sadly, [Elastic
+disagrees](https://opensource.org/node/1099).
 
-OpenSearch is an open source search and analytics engine derived from Elasticsearch 7.10.2, and is currently in an alpha state. While we're still in the process of getting set up, we are excited to build the best distributed search engine on the planet with you.
+**OpenSearch** is [a fork and spiritual successor](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)
+of [Elasticsearch 7.10.2](https://en.wikipedia.org/wiki/Elasticsearch) &
+[Kibana 7.10.2](https://en.wikipedia.org/wiki/Kibana), and we're looking to
+continue the work and culture that made us all cherish those technologies. We're
+committing to sustaining a high-performant, scalable and secure FOSS alternative,
+in the hopes that it will make the software community richer.
```

Comment:
At the risk of bringing something specific from Amazon, I found that we can express this better in tenets (or guiding principles). That list is [here](https://github.com/opensearch-project/opensearch#guiding-principles), and can evolve. Maybe we should just say that here? The principles are basically what you're saying, but more to the point.

Reply:
I'm not certain I understand: Your link points to this file, to a section that I've left unmodified in this PR.

Reply:
This: "we're looking to continue the work and culture that made us all cherish those technologies. We're committing to sustaining a high-performant, scalable and secure FOSS alternative, in the hopes that it will make the software community richer." reads to me like good intentions. What we need is mechanisms that ensure we actually do this, which can be expressed in tenets. So I am personally against making statements that are good intentions. What do you think @stockholmux?

Reply:
@dblock We haven't received a response for some time now. Do you want us to hold out a bit longer or may I begin addressing the review comments?

Reply:
My take on this is slightly different than @dblock. Some thoughts:

1. I'd rather not position OpenSearch as an _alternative_. Alternative implies always considering it a choice and worse, perhaps not the first choice.
2. I'm not sure the properties listed are useful - high performance, scalable, and secure come off a bit like marketing speak to me as they're largely unprovable as absolutes.

[Sorry for the radio silence. My github notifications are a bit of a disaster and I must have missed this.]

Reply:
I think what @stockholmux makes a lot of sense. I would trim/remove most of this.

Reply:
@stockholmux

Thank you for clarifying the situation. Apology accepted :-) For future reference, would you have preferred me to ping you after a couple of days?

(1) Is it the word choice of the concept that you're against? Conceptually, OpenSearch _is_ an alternative (specifically, an open-source alternative) and I'm not convinced it would be detrimental to acknowledge this. Many successful products list their competitors' products and highlight their own niche (their unique selling points, if you will).

If it's the former, I'm happy to accept suggestions from any native speaker :-)

(2) Those were taken from the website and I'm happy to update them, provided that we list something that differentiates us from Elasticsearch and Kibana. This could be just FOSS part, but is it a problem to formulating this as a vision?

Further, we could emphasis the **sustaining** part if we think that ES and Kibana had these qualities when we forked.

Reply:
@jesperolsson-se

I think I have a conceptual problem calling OpenSearch an alternative. It's a stronger position to define something in the affirmative. Saying OpenSearch is an FOSS alternative is a definition by contrast - meaningless if you don't already know Elasticsearch.

Additionally, OpenSearch and Elasticsearch will most likely diverge over time and grow into two different products in the same space. By framing it as an alternative here we're tying the project to the past, not the future.

Reply:
Also, always feel free to bug me if I'm not being responsive!

Reply:
@stockholmux 

You raise some good points and I definitely hadn't consider the latter. Generally speaking, I agree with you, but don't you think this position is a bit premature? I propose that, here and now, OpenSearch can only meaningfully be defined in contrast to Elasticsearch. The overwhelming majority of potential contributors and users will find this project _because_ it's an alternative and they are dissatisfied with the licence change. In a year or so, perhaps the situation is more akin to what you describe and I'd be happy to update the README accordingly, there and then.

My main concern (which prompted my ticket) is that we may not reach that scenario. There are many fascinating open source projects that beacon potential contributors, so how can we convince them not to walk away? OpenSearch has several factors stacked against our favour, and I believe the most trustworthy thing we can to is to humbly acknowledge them. For me, personally, it was [the statement from OSI](https://opensource.org/node/1099) that made me want to contribute to OpenSearch, but will only continue to do so as long as I view the project culture to align with my values. Choosing not to be up-front about our history and context is, to me, a step in the wrong direction (not malicious in any way, but problematic nonetheless).

I realise that this discussion has been on the long end and perhaps not really suited for a code review, so I'm open to cut it short. After you've given thought to my position, let me know yours and I'll update the PR accordingly. Should you wish to continue the discussion, I'd be happy to oblige but I won't try to "convince" you of my viewpoint. (And, of course, if you see a way forward that would resolve my worries about the future of the community, I'm eager to adapt.)

Reply:
There's a lot of good stuff in this commit, so I'd like to merge it as is, minus `, and we're looking tocontinue the work and culture that made us all cherish those technologies. We'recommitting to sustaining a high-performant, scalable and secure FOSS alternative,in the hopes that it will make the software community richer.` And in a future PR you can convince us that we should merge that text!

Reply:
I think I'm fine with @dblock's proposal.